routes/feedbackAndSupportRouter.js

```diff
@@ -39,4 +39,10 @@ router.get('/prepare-a-case', (req, res) => res.render('services/prepareACase',
 
 router.get('/workload-measurement-tool', (req, res) => res.render('services/workloadMeasurementTool', { backLinkUrl }))
 
+router.get('/check-rule39-mail', (req, res) =>
+  res.redirect(
```

Comment:
So, just to check... this page provides help text... something like

"The Check Rule 399 service does...  If you are having problems.. e-mail ..."?

The only issue with this is that if the service is down they won't see any help.

Reply:
Thanks @sp-watson , An excellent question 😄 
The page we are trying to redirect to is less of a "this is what the service does" type page, and more of a functional page with a form that allows them to submit a helpdesk (Zendesk) request

You make a valid point about what happens if the check rule 39 service is down. I'll check with the team ....